app/k8s_utils.py

```diff
@@ -20,7 +20,10 @@ def cordon_all_nodes(client, exclude_node_id: str):
     node_list = client.list_node()
     for node in node_list.items:
         logging.info("Inspecting node %s to cordon", node.metadata.name)
-        if node.metadata.labels.get("provisioner.cast.ai/node-id") != exclude_node_id:
+        skip=False
+        if node.metadata.labels.get("autoscaling.cast.ai/removal-disabled")=="true" and protect_eviction_disabled=="true":
```

Comment:
same as before, protect_eviction_disabled is not a string

Reply:
Made eviction_disabled a string and renamed it "removal_disabled"